modules/cnv-understanding-live-migration.adoc

```diff
@@ -5,16 +5,10 @@
 [id="cnv-understanding-live-migration_{context}"]
 = Understanding live migration
 
-Live migration is the process of moving a running virtual machine instance to 
-another node in the cluster without interruption to the virtual workload or 
-access. This can be a manual process, if you select a virtual machine instance 
-to migrate to another node, or an automatic process, if the 
-virtual machine instance has a `LiveMigrate` eviction strategy and the node on 
-which it is running is placed into maintenance. 
-
-[IMPORTANT]
```

Comment:
I would keep this admonition in this module as it's still relevant to understanding live migration.

Reply:
Kept. I went back and forth on keeping the admonition as I made it a prereq in the other module, but whatever you think will make it more noticeable.